astroML/density_estimation/xdeconv.py

```diff
@@ -208,9 +208,13 @@ def _EMstep(self, X, Xerr):
     def sample(self, size=1, random_state=None):
         if random_state is None:
             random_state = self.random_state
-        rng = check_random_state(random_state)
+        # I added a noqa here because I don't know if this function is
```

Comment:
No need for the comment, noqa is fine.

Reply:
I just wanted to point out something very strange is going there. The `check_random_state` is not statefull and the [example use](http://www.astroml.org/book_figures/chapter6/fig_XD_example.html) of `sample` makes me thing that `size` will only ever be a number. 

Removed comments, left noqa's.